resources/views/questions.blade.php

```diff
@@ -14,7 +14,7 @@
         <h1>Questions</h1>
         @foreach ($questions as $question)
 
-
+                    {{--TODO: intellij complains about the div in h3, find out why. --}}
```

Comment:
Obviously doesn't like a div inside of H3, but its the only way to get the look that it has right now without css. Should be @Incommensurable or @outofphasor's problem